File: Chapter_01_Introduction/Getting_Started_The_Easy_Problems/Easy/11799-Horror_Dash/Main.java
/*
 * Problem 11799 - Horror Dash
 * Java version 6/12/2016
 */

import java.io.PrintStream;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        StringBuffer results = new StringBuffer();

        int numberOfTestCases = input.nextInt();
        for (int i = 1; i <= numberOfTestCases; ++i) {
            int count = input.nextInt();
            int speed = 0;
            for (int j = 0; j < count; ++j) {
                int x = input.nextInt();
                if (x > speed) {
                    speed = x;
                }
            }
            results.append("Case " + i + ": " + speed + "\n");
        }
        System.out.print(results.toString());
    }
}

File: Chapter_01_Introduction/Getting_Started_The_Easy_Problems/Easy/11799-Horror_Dash/main.cpp
/*
 * Problem 11799 - Horror Dash
 */
#include <iostream>
#include <sstream>

int main() {
    std::ios::sync_with_stdio(0);
    std::cin.tie(0);

    std::ostringstream output;
    int test {};
    int testCases;
    std::cin >> testCases;
    while (testCases--) {
        int runners;
        std::cin >> runners;
        int speed {};
        for (int i = 0; i < runners; ++i) {
            int input;
            std::cin >> input;
            speed = std::max(input, speed);
        }
        output << "Case " << ++test << ": " << speed << '\n';
    }
    std::cout << output.str();
}
